### DISCUSSION TITLE
Why meta title and url is same?

### DISCUSSION THREAD

**Discussion by sudeep2003** [bagisto/bagisto#10156] [Q&A] [answered]
What is meta title and slug? What slug do? Why meta title and url is same?
I do not need the title and url will same, what can I do?

Here is the [screenshot](https://github.com/sudeep2003/screenshot/blob/main/abcd/Screenshot%20(541).png).

**Comment by amit-webkul** [comment]
Hello @sudeep2003 

### Meta Title
The **meta title** is an HTML element that defines the title of a web page. It appears in several key places, like Browser Tab, Search Engine Results, and Social Media Previews.

A well-crafted meta title helps with both SEO (search engine optimization) and user engagement, as it informs search engines and users about the content of the page.

### Slug
The **slug** is the part of a URL that comes after the domain name, typically representing the specific page or post. It is meant to be concise and descriptive, often derived from the page or post's title.

A good slug is short, readable, and often includes relevant keywords for SEO purposes. Slugs are typically lowercase and hyphenated to improve readability and SEO.

-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------

It is not mandatory to save the meta title and slug exactly as they are. You can customize them according to your specific needs and preferences.

**Reply by sudeep2003** [reply]
That's not happening, the URL is taking the title... Slug is not... that's what I saw.

**Reply by amit-webkul** [reply]
I can't understand what you are trying to say.

**Reply by sudeep2003** [reply]
@amit-webkul That's not happening, Meta Title is Being Used as URL Instead of Slug.

Look at the provided [screenshot](https://github.com/sudeep2003/screenshot/blob/main/abcd/Screenshot%20(541).png).

**Reply by suraj-webkul** [reply, marked as answer]
Hello @sudeep2003,

If you are talking about the SEO component for the products and category pages, it is currently just for demonstration purposes. However, the actual workflow is functioning as expected and works fine. We will fix this into the upcoming versions of Bagisto.

Please let me know if you need further clarification or assistance!

Best regards,  
Suraj
